CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   * Support Docker `forcePullImage` in job JSON definitions
   * `dcos job add` and `dcos job update` should show the help menu by default
 
+* Notes
+
+During this new minor release we also refactored the `dcos describe` subcommand from Python to Go.
+
```

Comment:
This is not in the right CHANGELOG section anymore.

Reply:
Fixed